src/ui/components/tenants/tenantDetail/thirdPartyPage/ThirdPartyPage.tsx

```diff
@@ -47,7 +47,7 @@ export const ThirdPartyPage = ({
 				<span>Back to tenant info</span>
 			</button>
 			<div className="third-party-section__cards">
-				<TenantDetailHeader onlyShowTenantId />
+				{/* <TenantDetailHeader onlyShowTenantId /> */}
```

Comment:
Please either remove it completely (incl the comment) or not, but don't leave it as a comment.